training/training/celeryconfig.py

```diff
@@ -0,0 +1,11 @@
+broker_url = "sqs://"  # "redis://localhost:6379"
+
+broker_transport_options = {
+    "predefined_queues": {
+        "training-queue.fifo": {
+            "url": "https://sqs.us-east-1.amazonaws.com/521654603461/training-queue.fifo",
```

Comment:
is `celeryconfig.py` meant to encode any hardcoded values?

Reply:
That url should stay static as long as the terraform queue name in sqs.tf doesn't get modified, I don't know if there's a way to not hardcode it

Reply:
understood.